tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -55,7 +56,7 @@ describe('Effective Date Time component', () => {
     valid: false,
     isFutureEffective: true,
     // effective date is more than 10 days from now
```

Comment:
Move comment up please (so it makes sense).

Reply:
👍

Reply:
Done now.